apps/artist/client/router.coffee

```diff
@@ -63,8 +64,8 @@ module.exports = class ArtistRouter extends Backbone.Router
   overview: ->
     @view = new OverviewView @options
     $('body').append @jump.$el
-    @view.on 'artist:overview:sync', =>
-      attachCTA @model
+    @view.on 'artist:overview:sync', (artist) =>
+      attachCTA new Artist(_.extend({}, artist, @model.attributes))
```

Comment:
The reason we want to use the initial bootstrapped server-side `@model`, is it has the current artist page's info (name and cover image), plus is used for the current black bar CTA.